### DISCUSSION TITLE
How the front page of the Demo is made？

### DISCUSSION THREAD

**Discussion by Evolution-detector** [squidfunk/mkdocs-material#3946] [Q&A] [answered]
I'm still new to programming and after successfully installing Material for MkDocs, only my articles are displayed locally. I would like to design a cool home page for my personal website, like the author's demo.Can anyone tell me how to configure this page?
![image](https://user-images.githubusercontent.com/64944480/169948171-37c9504c-85f3-4f41-8e9c-b1dce2b28fb8.png)
I'm not a native English speaker, so I'm not very efficient at searching for information. If there are already answers in the discussion forum, I'm sorry to bother you, but please send me a link to the relevant answer. Thank you.

**Comment by squidfunk** [comment, marked as answer]
The source code for the new landing page is contained in the [Insiders repository](https://squidfunk.github.io/mkdocs-material/insiders/) that you can get access to by [sponsoring me](https://squidfunk.github.io/mkdocs-material/insiders/#how-to-become-a-sponsor). It will always stay in Insiders, mainly because I cannot share the source images openly due to licensing restrictions. It's a custom template.

**Reply by Evolution-detector** [reply]
Thank you for your very prompt reply. I was wondering if this page is configurable via a yml file? Or is that a traditional page, and modifying the content inside (like replacing the background image) requires me to understand the syntax of js, css, html, etc.

**Reply by squidfunk** [reply]
It's an entirely custom template, so you'd need to know some CSS and HTML to adapt it.
